lib/mayaUsd/ufe/wrapUtils.cpp

```diff
@@ -219,43 +196,8 @@ int ufePathToInstanceIndex(const std::string& ufePathString)
 
 bool isAttributeEditAllowed(const PXR_NS::UsdAttribute& attr)
 {
-    // get the property spec in the edit target's layer
-    const auto& prim = attr.GetPrim();
-    const auto& stage = prim.GetStage();
-    const auto& editTarget = stage->GetEditTarget();
-
-    // get the index to edit target layer
-    const auto targetLayerIndex = findLayerIndex(attr, editTarget.GetLayer());
-
-    // get the strength-ordered ( strong-to-weak order ) list of property specs that provide
-    // opinions for this property.
-    const auto& propertyStack = attr.GetPropertyStack();
-
-    // HS March 15th,2021
-    // TODO: This code works as long as existing opinions are in the stage’s local LayerStack.
-    // Relying on the "expanded primIndex" would be a better way to detect the opinions across
-    // composition arc(s). Some more details can be found:
-    // https://groups.google.com/g/usd-interest/c/xTxFYQA_bRs/m/lX_WqNLoBAAJ
-    PXR_NS::SdfLayerHandle strongLayer;
-    for (const auto& spec : propertyStack) {
-
-        // skip if the edit target layer is stronger than the propSpec layer
-        const auto propSpecLayerIndex = findLayerIndex(attr, spec->GetLayer());
-        if (targetLayerIndex <= propSpecLayerIndex) {
-            continue;
-        }
-
-        if (spec) {
-            strongLayer = spec->GetLayer();
-            break;
-        }
-    }
-
-    if (strongLayer) {
-        return false;
-    }
-
-    return true;
+    std::string dummyStr;
+    return ufe::isAttributeEditAllowed(attr, &dummyStr);
```

Comment:
That's what I tried first but boost python binding doesn't like it. 

```
 Stack trace:
15:   mayaUsd.dll!std::basic_string<char,std::char_traits<char>,std::allocator<char> >::assign Line 2659
15:   mayaUsd.dll!MayaUsd_v0::ufe::isAttributeEditAllowed Line 435
15:   _ufe.pyd!boost::python::objects::caller_py_function_impl<boost::python::detail::caller<bool (__cdecl*)(pxrInternal_v0_21__pxrReserved__::UsdAttribute const &),boost::python::default_call_policies,boost:
```

Reply:
That's very strange, I think the fact you have Boost in the call stack is simply because it's being called from Python.  This should work, let me know if you'd like us to debug this together.